Apps/Admin/Client/Services/IDashboardApi.cs

```diff
@@ -0,0 +1,72 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+
+namespace HealthGateway.Admin.Client.Services;
+
+using Refit;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+/// <summary>
+/// API to fetch the Dashboard from the server.
+/// </summary>
+public interface IDashboardApi
+{
+    /// <summary>
+    /// Retrieves the count of registered users.
+    /// </summary>
+    /// <returns>HttpResponseMessage.</returns>
+    /// <param name="timeOffset">The offset from the client browser to UTC.</param>
+    [Get("/RegisteredCount")]
+    Task<HttpResponseMessage> GetRegisteredUserCount(int timeOffset);
```

Comment:
Are we returning raw data for all of these methods or JSON?  If its JSON we should be using a typed method instead of HttpResposneMessage